2-structured-data/config.py

```diff
@@ -51,15 +51,19 @@
 #   $ cloud_sql_proxy -instances=your-connection-name=tcp:3306
 #
 # Alternatively, you could use a local MySQL instance for testing.
+# Port 3306 is the standard MySQL port. If you need to use a different port,
+# change the 3306 to a different port number.
```

Comment:
The comment seems a little strange. Given this is all CloudSQL, The port should always be 3306, how would they use a different port with CloudSQL?